benchmarks/pom.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
A general request - please avoid stacking several orthogonal changes in a single pull request
it makes it harder to review and focus on the relevant changes to check

Reply:
You are right